src/Rpc/RpcServer.cpp

```diff
@@ -655,7 +654,7 @@ bool RpcServer::on_get_info(const COMMAND_RPC_GET_INFO::request& req, COMMAND_RP
   uint64_t sizeMedian = Common::medianValue(blocksSizes);
   uint64_t nextReward = 0;
   int64_t emissionChange = 0;
-  if (!m_core.getBlockReward(res.block_major_version, sizeMedian, 0, alreadyGeneratedCoins, 0, nextReward, emissionChange)) {
+  if (!m_core.getBlockReward(res.block_major_version, sizeMedian, 0, res.already_generated_coins, 0, nextReward, emissionChange)) {
```

Comment:
This will make incompatible with old GUI wallets ran on same machine: if daemon with this change is launched first and GUI wallet next, GUI will run with it's builtin node instead of running in rpc mode via this daemon.

Reply:
We have added another breaking changes before so this one doesn't matter now, after several updates everyone will be on same page.